src/rgw/rgw_file.cc

```diff
@@ -1834,7 +1834,7 @@ namespace rgw {
     ceph_assert(! dlo_manifest);
     ceph_assert(! slo_info);
 
-    perfcounter->inc(l_rgw_put);
+    rgw::op_counters::global_op_counters->inc(l_rgw_op_put);
```

Comment:
these shouldn't rely on the global variable. can this not use `rgw::op_counters::inc()`?